src/clients/paymentsRefunds/get/get.spec.ts

```diff
@@ -0,0 +1,31 @@
+import get from '.';
+import { RestClient } from '@utils/restClient';
+import { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+import type { Get } from './types';
+
+jest.mock('@utils/restClient');
+
+describe('Testing payments refunds, get', () => {
+	test('should find a refund by refund_id ', async () => {
```

Comment:
```suggestion
	test('should include the refund_id in the path of the url ', async () => {
```
essa sugestão é porque a descrição que tinha antes não condiz com o que o teste verifica. Ser capaz de encontrar um refund por id faria mais sentido num teste de integração.

Reply:
Perfeito! Obrigada, Romero :)